iib/workers/tasks/build_create_empty_index.py

```diff
@@ -58,21 +59,21 @@ def _get_present_operators(from_index, base_dir):
 @app.task
 @request_logger
 def handle_create_empty_index_request(
-    from_index, request_id, binary_image=None, labels=None, binary_image_config=None
+    from_index, request_id, fbc=False, binary_image=None, labels=None, binary_image_config=None
 ):
     """Coordinate the the work needed to create the index image with labels.
 
     :param str from_index: the pull specification of the container image containing the index that
         the index image build will be based from.
     :param int request_id: the ID of the IIB build request
+    :param bool fbc: specifies whether a File Based Catalog index image should be created
```

Comment:
I am not native speaker so I am not sure what is right `File Based Catalog` or `File-Based Catalog` with `-` sign?

Reply:
Changed it already